static/styles/left-sidebar.scss

```diff
@@ -259,10 +259,12 @@ li.top_left_starred_messages {
     }
 }
 
-.top_left_private_messages i.fa-envelope {
-    position: relative;
-    top: -1px;
-    font-size: 11px;
+.top_left_private_messages {
+    i.fa-envelope {
+        position: relative;
+        top: -1px;
+        font-size: 11px;
+    }
```

Comment:
No need to nest CSS rulesets in such cases, nesting should be done where the parent have their own style and a child element with different style or if parent has no style and has more than 1 child element.

Here the `.top_left_private_messages` doesn't have have its own style and only has one child element. So no need to nest them. We can nest them later when we'd have more sub elements.

Reply:
I thought of doing the nesting now as this PR is refactoring the whole CSS(left-sidebar.scss) file. I guess it will not be good to make a separate commit just for nesting (for nesting elements like this which has any CSS changes in future) in the future.